diff --git a/sdk/storage/azure-storage-common/vcpkg/Config.cmake.in b/sdk/storage/azure-storage-common/vcpkg/Config.cmake.in
index e10387a..874dc5a 100644
--- a/sdk/storage/azure-storage-common/vcpkg/Config.cmake.in
+++ b/sdk/storage/azure-storage-common/vcpkg/Config.cmake.in
@@ -7,11 +7,6 @@ include(CMakeFindDependencyMacro)
 find_dependency(Threads)
 find_dependency(azure-core-cpp)
 
-if(NOT WIN32)
-  find_dependency(LibXml2)
-  find_dependency(OpenSSL)
-endif()
-
 include("${CMAKE_CURRENT_LIST_DIR}/azure-storage-common-cppTargets.cmake")
 
 check_required_components("azure-storage-common-cpp")
